tools/kagami/src/config.rs

```diff
@@ -64,16 +64,31 @@ mod client {
 }
 
 mod peer {
+    use std::path::PathBuf;
+
     use iroha_config::iroha::ConfigurationProxy as IrohaConfigurationProxy;
 
     use super::*;
 
-    #[derive(ClapArgs, Debug, Clone, Copy)]
-    pub struct Args;
+    #[derive(ClapArgs, Debug, Clone)]
+    pub struct Args {
+        /// Specifies the value of `genesis.file` configuration parameter.
+        ///
+        /// Note: relative paths are not resolved but included as-is.
+        #[arg(long)]
+        genesis_file_in_config: Option<PathBuf>,
```

Comment:
What about `submit-genesis` (in accordance to `iroha` cli) or `include-genesis`?

Reply:
- `submit-genesis` doesn't really communicate that the value of this argument is the path of the `genesis.file` parameter that will be used as is
- `include-genesis` tells more that Kagami will read the path and include genesis into the config file

(IMO)

So, I chose the same approach as with `kagami genesis --executor-path-in-genesis <PATH>`